doc/how_to_contribute/how_to_contribute_to_site.rst

```diff
@@ -0,0 +1,73 @@
+How-To Contribute to This Site
+==============================
+
+This explains how to make and submit changes to this site.
+This how-to guide assumes you are on Ubuntu 20.04.
+
+Setting up your development environment
+---------------------------------------
+
+Follow the tutorial :doc:`/doc/tutorials/getting_started/getting_started` to set up your workspace for testing local changes.
+
+All further instructions assume you are the directory containing a local checkout of ``moveit2_tutorials``.
+
+Build and view the website locally
+----------------------------------
+
+To build the website locally run this command:
+
+.. code-block:: bash
+
+  make html
+
+Then you can use this to open the site in your webbrowser:
+
+.. code-block:: bash
+
+  open ./build/html/index.html
```

Comment:
```suggestion
Then you can open the local build of the site in your web browser from: ``/build/html/index.html``. For example, to open the local site in Firefox the command would be:

.. code-block:: bash

  firefox ./build/html/index.html
```

Reply:
I'd put a . or remove the first slash from the instructions you added.  Does that make sense?  The build directory isn't at the root of the filesystem.

Reply:
I've included this change.